backend/src/services/integrationService.ts

```diff
@@ -554,15 +554,67 @@ export default class IntegrationService {
     return IntegrationRepository.findGlobalIntegrationsStatusCount(args, this.options)
   }
 
+  private decryptIntegrationSettings(platform: string, settings: any): any {
```

Comment:
Can you perhaps just refactor this to a file in `common_services` lib? Eventually I want to move all the `integrationService.ts` there because there are lots of cases where we need to call these functions from `services` libs and apps and it's just easier if we start doing this with new things. You can just call it `integration.service.ts` and put it in `common_services` lib and just move the functions you added (related to encrypt/decrypt etc.) there not everything of course. Whatever is easy to move and doesn't take too much time :). The encrypt/decrypt from utils can also be moved to `common` library.